recipe/patches/3.16.2/0002-find_-Use-debug-logging-infrastructure.patch

```diff
@@ -1,945 +0,0 @@
-From 9435c0e694a0d2bea47aa18b7643827742dad874 Mon Sep 17 00:00:00 2001
-From: Robert Maynard <robert.maynard@kitware.com>
-Date: Wed, 11 Dec 2019 13:01:09 -0500
-Subject: [PATCH 2/3] find_*: Use debug logging infrastructure
-
-Teach the find_package, find_library, find_program, find_path, and
-find_file commands to print debug log messages when enabled by the
-`--debug-find` command-line option or `CMAKE_FIND_DEBUG_MODE` variable.
```

Comment:
My guess is that these patches have been included in https://gitlab.kitware.com/cmake/cmake/-/merge_requests/3935?commit_id=204b8d9f4e9c5a64e1fa6a0ee4e6dc2911694bae